pkg/tsdb/cloudmonitoring/time_series_filter_test.go

```diff
@@ -386,6 +388,77 @@ func TestTimeSeriesFilter(t *testing.T) {
 		require.True(t, ok)
 		assert.Equal(t, "114250375703598695", labels["resource.label.instance_id"])
 	})
+
+	t.Run("parseResponse successfully parses metadata for distribution valueType", func(t *testing.T) {
+		t.Run("exponential bounds", func(t *testing.T) {
+			data, err := loadTestFile("./test-data/3-series-response-distribution-exponential.json")
+			require.NoError(t, err)
+			assert.Equal(t, 1, len(data.TimeSeries))
+
+			res := &backend.DataResponse{}
+			require.NoError(t, (&cloudMonitoringTimeSeriesFilter{}).parseResponse(res, data, "test_query"))
+
+			require.NotNil(t, res.Frames[0].Meta)
+			assert.Equal(t, sdkdata.FrameMeta{
+				ExecutedQueryString: "test_query",
+				Custom: map[string]interface{}{
+					"groupBys":        []string(nil),
+					"alignmentPeriod": "",
+					"labels": map[string]string{
+						"resource.label.project_id": "grafana-prod",
+						"resource.type":             "https_lb_rule",
+					},
+					"perSeriesAligner": "",
```

Comment:
I was thinking about adding a separate test (out of scope of this PR, I'm *not* at all asking you to add it) which asserts the values which are empty here (`groupBys`, `alignmentPeriod`, `perSeriesAligner`) in order to ensure the rest of the metadata is successfully parsed for distribution valueType.
Do you think that information is realistic/useful here?

Reply:
I added check for `groupBys`.

I'm not sure adding `alignmentPeriod` and `perSeriesAligner` is useful.
https://github.com/grafana/grafana/blob/v8.4.1/pkg/tsdb/cloudmonitoring/time_series_filter.go#L147-L148
These are getting from `Params` field, setting these params is deeply depend on implementation.